src/display.rs

```diff
@@ -388,6 +388,8 @@ impl Display {
             let height = psize.height as f32;
             let cell_width = self.size_info.cell_width;
             let cell_height = self.size_info.cell_height;
+            let cols = self.size_info.cols().0;
+            let lines = self.size_info.lines().0;
```

Comment:
I think naming this something like `old_*` or `previous_*` instead of just cols/lines would make it a little more obvious what's going on here.